lib/prepare.js

```diff
@@ -84,6 +83,13 @@ async function createDirectories(folderName, baseDirectory) {
     await fse.mkdirs(directories.in);
     await fse.mkdirs(directories.out);
 
+    if (!pipeline) {
```

Comment:
I would rather not try to make prepare in the pipeline compatible with the SDK. this will make it more difficult when we remove the `worker` class from the SDK and switch fully to the pipeline

the pipeline should not need to know about `postprocessing` or non pipeline cases